rust/otap-dataflow/crates/otap/src/otlp_receiver.rs

```diff
@@ -39,6 +42,13 @@ pub const OTLP_RECEIVER_URN: &str = "urn:otel:otlp:receiver";
 pub struct Config {
     listening_addr: SocketAddr,
     compression_method: Option<CompressionMethod>,
+    /// Maximum number of concurrent in-flight requests (default: 1000)
+    #[serde(default = "default_max_slots")]
+    max_slots: usize,
```

Comment:
Shouldn't we name this parameter in a way that better reflects what's mentioned in the comment?
Something like `max_concurrent_in_flight_requests`?